src/js/parsers/section.js

```diff
@@ -204,7 +238,18 @@ class SectionParser {
       this._createMarker();
     }
 
-    sections.push(state.section);
+    // push listItems onto the listSection or add a new section
+    if (state.section.isListItem && lastSection && lastSection.isListSection) {
+      lastSection.items.append(state.section);
```

Comment:
Note to self - add test to ensure consecutive `<ul>`'s do not get merged together.

Reply:
Added test + fix